src/Constraint/JsonValueMatchesMany.php

```diff
@@ -66,4 +66,23 @@ protected function matches($other): bool
         }
         return true;
     }
+
+    /**
+     * Returns a string representation of matches that evaluate to false.
+     * 
```

Comment:
This will make codeclimate stop complaining:

```suggestion
     *
```